slashing-protection/src/integration-test/java/tech/pegasys/web3signer/slashingprotection/InterchangeExportIntegrationTest.java

```diff
@@ -109,8 +84,6 @@ void canCreateDatabaseWithEntries() throws IOException, URISyntaxException {
     slashingProtection.export(exportOutput);
     exportOutput.close();
```

Comment:
exportOutput.close (I know it exists before your changes), shouldn't it be part of try/resources ?

Reply:
not sure how. I.e. I need to close the stream, but still keep a handle to the object such that I can extract the string from it - and a try-with-resources will both close, and destoy the object at the same time, won't it?